docs/release-process.md

```diff
@@ -0,0 +1,102 @@
+# svcwatch Release Process
+
+## Preparation
+
+Currently there is no dedicated branch for releases. svcwatch is simple enough,
+has few dependencies, and we're not planning on doing backports. Therefore
+we apply release tags to the master branch.
+
+### Tagging
+
+Example:
+```
+git checkout master
+git pull --ff-only
+git tag -a -m 'Release v0.3' v0.3
+```
+
+This creates an annotated tag. Release tags must be annotated tags.
+
+### Build
+
+Perform a final check that the image is built correctly. First, ensure that
+base images are up-to-date:
+
+```
+podman pull docker.io/golang:1.18
+podman pull registry.access.redhat.com/ubi8/ubi-minimal:latest
+```
+
+Note that the base images are expected to change over time. Make sure
+you pull the images that are used by the project's Containerfile.
+
+
+Build the image by running `make image-build`.
+
+Double check that the build contains the proper git hash and tag. This can be
+done by running the container and looking at the initial logging the svcwatch
+binary produces.
+
+### Image Push
+
+Apply a temporary pre-release tag to the image:
+```
+podman tag quay.io/samba.org/svcwatch:latest quay.io/samba.org/svcwatch:v0.3pre1
+```
+
+Log in to quay.io. Example:
+```
+podman login quay.io
+```
+
+Push the image using the temporary pre-release tag. Example:
+```
+podman push quay.io/samba.org/svcwatch:v0.3pre1
+```
+
+Wait for the security scan to complete. There shouldn't be any issues if you
+properly updated the base images before building. If there are issues and you
+are sure you used the newest base images, check the base images on quay.io and
+make sure the number of issues are are identical. The security scan can take
```

Comment:
```suggestion
make sure that number of issues are identical. The security scan can take
```

Reply:
The double are needs to be fixed. But I don't think that the other part of the fix is grammatical. I think "that the" would be OK though.

Reply:
Updated